readme.md

```diff
@@ -46,16 +46,28 @@ example, or keep reading for more of a guide.
 
 ### Getting it going
 
-First, import the library. Copy the file [`grayscale.py`](./grayscale.py) to
-your project directory and import like so:
+First, you have to import the library.
 
-```python
-# Fix import path so it finds the grayscale library
-import sys
-sys.path.insert(0, "/".join(__file__.split("/")[0:-1]))
+We're working on getting the grayscale library added to the standard Thumby
+library, but for now you have to do some manual work. You can either:
+
+1. manually add [`thumbyGrayscale.py`](./lib/thumbyGrayscale.py) to the `/lib`
+   directory on your Thumby (and require your users to do the same), or;
+2. supply the library with your game by copying
+   [`thumbyGrayscale.py`](./lib/thumbyGrayscale.py) to your game folder and
+   setting the import path:
+
+  ```python
+  # Fix import path so it can find a local grayscale library if present
+  if '__file__' in globals():
```

Comment:
If someone has copied thumbyGrayscale.py to their game dir, will an import work with Fast Execute (i.e. where the sys.path adjustment won't happen)?
While the auto-dirname approach is nice, it might just be cleaner to suggest doing
```
syspath.insert('/Games/YourGameName')
```
or just
```
import Games.YourGameName.thumbyGrayscale
```